### PR TITLE
rework AbstractPlotting tests

### PR DESCRIPTION
**Pull request by asinghvi17**
Requires JuliaPlots/MakieGallery.jl#49 first.

### CODE REVIEW

**Comment by asinghvi17** [comment]
```julia
(v1.1) pkg> test AbstractPlotting
   Testing AbstractPlotting
 Resolving package versions...
    Status `/var/folders/gl/tdt9v5vj4xqg5cdsz0ll1j7h0000gn/T/tmpLYnQmq/Manifest.toml`
  [621f4979] AbstractFFTs v0.4.1
  [...]
  [4ec0a83e] Unicode  [`@stdlib/Unicode`]
Test Summary: | Pass  Total
Quaternions   |    8      8
┌ Warning: Package MakieGallery does not have Test in its dependencies:
│ - If you have MakieGallery checked out for development and have
│   added Test as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with MakieGallery
└ Loading Test into MakieGallery from project dependency, future warnings for MakieGallery are suppressed.
Skipping Comparing contours, image, surfaces and heatmaps
(removed from tests explicitly)
Test Summary:       | Pass  Total
Gallery short tests |   42     42
   Testing AbstractPlotting tests passed
```

File: test/runtests.jl [hunk 1]
```diff
@@ -4,79 +4,90 @@ using Test
...
-   # skip if the title is in the list of exceptions
-  if database[i].title ∈ exc_str
+    # Load only short tests
 
-     print("Skipping " * database[i].title * "\n(removed from tests explicitly)\n")
+    database = MakieGallery.load_tests()
 
-     continue  # skip the entry
+    # This one is broken, all the others are fine.  I don't know why this fails.  It doesn't fail on my machine :P
+    exc_str = Set(["Comparing contours, image, surfaces and heatmaps"])
```
Comment (asinghvi17) [inline test/runtests.jl]: @SimonDanisch any reason this would fail `@test_nowarn`?

**Comment by asinghvi17** [comment]
CI fail is unrelated; a new MakieGallery version needs to be released, then it should work.

**Comment by asinghvi17** [comment]
@SimonDanisch LGTM?

**Comment by SimonDanisch** [comment]
Yes, thank you!! :)  although I'd like to have it the other way around - first tag MakieGallery, and then AbstractPlotting without checking out MakieGallery#master...

**Comment by asinghvi17** [comment]
Fair enough (and we could get rid of this afterwards) but right now it's needed to escape the ImageCore incompatibility problem.

**Pull request merged**